blog/convert-raw-trace-data/index.qmd

```diff
@@ -0,0 +1,223 @@
+| 23             | TRD_MDFR_SRO_CD      | TEXT                  |             | 
+| 24             | rpt_side_cd          | TEXT                  | B           | 
+| 25             | BUYER_CMSN_AMT       | REAL                  | 10.00       | 
+| 26             | BUYER_CPCTY_CD       | TEXT                  | A           | 
+| 27             | SLLR_CMSN_AMT        | REAL                  | 12.00       | 
+| 28             | SLLR_CPCTY_CD        | TEXT                  | 12345       | 
+| 29             | RPT_PRTY_ID          | TEXT                  | D           | 
+| 30             | cntra_mp_id          | TEXT                  | D           | 
+| 31             | CNTRA_PARTY_GVP_ID   | TEXT                  |             | 
+| 32             | LCKD_IN_FL           | TEXT                  |             | 
+| 33             | ATS_FL               | TEXT                  |             | 
+| 34             | spcl_trd_fl          | TEXT                  |             | 
+| 35             | TRDG_MKT_CD          | TEXT                  | S1          | 
+| 36             | PBLSH_FL             | TEXT                  | N           | 
+| 37             | SYSTM_CNTRL_DT       | TEXT                  | 20130102    | 
+| 38             | SYSTM_CNTRL_NB       | TEXT                  | 2000000003  | 
+| 39             | pr_trd_dt            | TEXT                  | 20130103    | 
+| 40             | PREV_TRD_CNTRL_NB    | TEXT                  |             | 
+| 41             | FIRST_TRD_CNTRL_DT   | TEXT                  | 20130105    | 
+| 42             | FIRST_TRD_CNTRL_NB   | INTEGER               | 2           | 
+| 43             | msg_seq_nb           | TEXT                  | AC_2000000003 | 
+| 44             | orig_msg_seq_nb      | TEXT                  | AC_2        | 
+| 45             | days_to_sttl_ct      | INTEGER               | 4 (days between stlmnt_dt and trd_exectn_dt) | 
+
+: Structure of the local SQLite database {#tbl-finaltable}
+
+## Create a local SQLite from downloaded TRACE data
+
+After successfully ordering and downloading the data, you should possess all necessary raw files to start creating a SQLite. In the following, we assume that you possess a folder with all downloaded TRACE data (enhanced and academic) called "TRACE_Downloads", with a similar structure as follows:
+```
+TRACE_Downloads/
+├── academic_txts/
+│   ├── 0047-corp-academic-trace-data-2013-01-02.txt
+│   ├── 0047-corp-academic-trace-data-2013-01-03.txt
+│   └──...
+├── EHDwC 2020.zip
+├── EHDwC 2021.zip
+├── EHDwC 2022.zip
+├── ...
+```
+
+The below R code allows the user to convert the raw data provided by FINRA (Academic Data as well as Enhanced Historical Data) into a SQLite for further analyses. For the curious reader, you can find implementation details and code documentation in our [GitHub Repository](https://github.com/DerKevinRiehl/bondTradingDataSQLiteConverterR). 
+
```

Comment:
The link does not work

Reply:
Hi Christoph, repo is changed to public now.
https://github.com/DerKevinRiehl/bondTradingDataSqliteDbConverterR

Reply:
Link corrected in article at all occurences to our Github Repo